internal/services/monitor/monitor_data_collection_rule_resource.go

```diff
@@ -763,9 +763,37 @@ func (r DataCollectionRuleResource) Arguments() map[string]*pluginsdk.Schema {
 									MinItems: 1,
 									Elem: &pluginsdk.Schema{
 										Type: pluginsdk.TypeString,
-										ValidateFunc: validation.StringInSlice(
-											datacollectionrules.PossibleValuesForKnownSyslogDataSourceFacilityNames(),
```

Comment:
Rather than overriding this list, we need the Swagger updated to include these new values.

Whilst the API _as a whole_ may support the new values, since the Swagger isn't including those values - it's likely that these are only supported in a new version of the API - so we need to update the API Definition to account for that.

As such would you mind sending a PR to add these to the applicable Swagger definition, so that the updated values can be piped through into `datacollectionrules.PossibleValuesForKnownSyslogDataSourceFacilityNames`? Thanks!